tests/Unit/ExamplePhpunitTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Tests\Unit;
+
+use PHPUnit\Framework\TestCase;
+
+class ExamplePhpunitTest extends TestCase
+{
+    /**
+     * A basic test example.
+     *
+     * @return void
+     */
+    public function testBasicTest()
+    {
+        $this->assertTrue(true);
+    }
```

Comment:
Just a heads up that PHPUnit also supports the following methods of declaring a method to be tests (they have to be public of course):

```suggestion

use PHPUnit\Framework\Attributes\Test;
use PHPUnit\Framework\TestCase;

class ExamplePhpunitTest extends TestCase
{
    public function testBasicTest(): void
    {
        $this->assertTrue(true);
    }
    
    /** @test */
    public function itAssertsTrue(): void
    {
        $this->assertTrue(true);
    }
    
    #[Test]
    public function itAssertsTrueAgain(): void
    {
        $this->assertTrue(true);
    }
```

Reply:
I've gone ahead and committed this suggestion, but I am reasonably certain that this branch won't work with the latter two at the moment. It may detect - and even run - the tests, but I don't think results will be matched properly off-hand